src/client/styles/components/_button.scss

```diff
@@ -15,12 +15,12 @@
   box-shadow: none;
   display: block;
   float: left;
-  line-height: 56px;
+  //line-height: 56px;
```

Comment:
I have few commented code kept for my reference.
Once all the changes are done for nested steps, I will remove the comments.